docs/en/Configuring-Alluxio-with-MapR-FS.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+title: Configuring Alluxio with MapR-FS
+nickname: Alluxio with MapR-FS
+group: Under Store
+priority: 3
+---
+
+This guide describes how to configure Alluxio with [MapR-FS](https://www.mapr.com/products/mapr-fs) as the under
+storage system.
+
+# Compiling Alluxio with MapR Version
+
+Alluxio must be [compiled](Building-Alluxio-Master-Branch.html) with the correct MapR distribution to integrate with
+MapR-FS. Here are some values of `hadoop.version` for different MapR distributions:
+
+<table class="table table-striped">
+<tr><th>MapR Version</th><th>hadoop.version</th></tr>
+<tr>
+  <td>5.2</td>
+  <td>2.7.0-mapr-1607</td>
+</tr>
+<tr>
+  <td>5.1</td>
+  <td>2.7.0-mapr-1602</td>
+</tr>
+<tr>
+  <td>5.0</td>
+  <td>2.7.0-mapr-1506</td>
+</tr>
+<tr>
+  <td>4.1</td>
+  <td>2.5.1-mapr-1503</td>
+</tr>
+<tr>
+  <td>4.0.2</td>
+  <td>2.5.1-mapr-1501</td>
+</tr>
+<tr>
+  <td>4.0.1</td>
+  <td>2.4.1-mapr-1408</td>
+</tr>
+</table>
+
+# Configuring Alluxio for MapR-FS
+
+Once you have compiled Alluxio with the appropriate `hadoop.version` for your MapR distribution, you have to configure
+Alluxio to recognize the MapR-FS scheme and URIs. Alluxio can use the HDFS client to access MapR-FS. In order to enable
+the HDFS client to access MapR-FS URIs, you have to add the URI prefix `maprfs:///` to the configuration variable
+`alluxio.underfs.hdfs.prefixes`.
+
+```
+alluxio.underfs.hdfs.prefixes=hdfs://,glusterfs:///,maprfs:///
+```
+
+This configuration parameter should be set for all the Alluxio servers (masters, workers). Please read how to
+[configure Alluxio](Configuration-Settings.html).
+
+This parameter should be set whenever you access Alluxio. This means the parameter should be set for any application
+(MapReduce, Spark, Flink, etc.) that accesses Alluxio.
```

Comment:
Perhaps combine this paragraph with the previous and add some instruction about how to set it for the clients?